### PR TITLE
fix(*): use shared signed extensions to prevent StoreCallMetadata warning

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

Prevents a warning from being logged by registering shared signed extensions

### Breaking Changes

None

### JIRA Link

[DA-482](https://polymesh.atlassian.net/browse/DA-482)

### Checklist

- [ ] Updated the Readme.md (if required) ?

[DA-482]: https://polymesh.atlassian.net/browse/DA-482?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ Monorepo for Polymesh SDK compatible Signing Managers.
 - Testing: `yarn test <packageName>`
 - Building: `yarn build <packageName>`
 
+If using on a unix type computer, all packages can be built (or test/lint) with:
+
+```sh
+ls packages | cut -d "_" -f 1 | xargs -I {} bash -c "yarn build} {}"
```
Comment (polymath-eric) [inline README.md:23]: I thought this was useful, but probably won't work on Windows, so I didn't want to add it to the "scripts"

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**